digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -549,7 +550,9 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
     val targetLinks = targets.flatMap(rd => {
       getUniqueRoadAddressLink(rd.toLong)
     })
-    transferRoadAddress(sourceLinks, targetLinks, user)
+    val transferredRoadAddresses = transferRoadAddress(sourceLinks, targetLinks, user)
```

Comment:
If one checks the RoadAddressServiceSpec class, line 405:
`      val newLinks = roadAddressService.transferRoadAddress(floatingLinks, targetLinks, User(1L, "foo", new Configuration()))
`
The transferRoadAddress is considered in the test case "transferRoadAddress should keep calibration points".